docs/Extra_compute_drac.rst

```diff
@@ -386,11 +386,44 @@ Canada Documentation <https://docs.alliancecan.ca/wiki/Cedar>`__.
   they start.
 
 
+Narval
+^^^^^^
+
+Narval is a cluster located at the ÉTS (École de Technologie Supérieure) in
+Montreal. It uses SLURM to schedule jobs. Its full documentation can be found `here
+<https://docs.alliancecan.ca/wiki/Narval>`__, and its current status `here
+<http://status.alliancecan.ca>`__.
+
+You can access Narval via ssh:
+
+.. prompt:: bash $
+
+    ssh <user>@narval.computecanada.ca
+
+Where ``<user>`` is the username you created previously (see `Account Creation`_).
+
+While Narval has a filesystem organization similar to the above clusters, and the
```

Comment:
Shouldn't we also put emphasis on the fact that Narval is possibly the best adapted of all clusters for LLM-scale by the fact that nodes are IB-interconnected + n x nvlinked A100 ?